src/modeles/autorisations/autorisationBase.js

```diff
@@ -7,6 +7,7 @@ class AutorisationBase extends Base {
 
     this.permissionAjoutContributeur = false;
     this.permissionSuppressionContributeur = false;
+    this.droitsProprietaires = false;
```

Comment:
Que penses-tu de rester dans le pattern en place, et renommer ça `this.permissionSuppressionService` ?